packages/react-components/react-select/src/stories/Select/SelectAppearance.stories.tsx

```diff
@@ -1,8 +1,6 @@
 import * as React from 'react';
 import { Select } from '@fluentui/react-select';
```

Comment:
Should `Select` be imported from `@fluentui/react-components/lib/unstable`?

Reply:
It should be fine to keep importing from the current package until `react-select` is moved to stable. That way this import doesnt have to keep getting updated when a package goes from development -> unstable -> stable.